frontend/src/routes/ClusterManagement/Clusters/ClusterDetails/ClusterOverview/ClusterOverview.tsx

```diff
@@ -32,12 +33,25 @@ export function ClusterOverviewPageContent(props: { canGetSecret?: boolean }) {
     return (
         <AcmPageContent id="overview">
             <PageSection>
+                {cluster?.statusMessage && (
+                    <AcmAlert
+                        isInline
+                        title={t(`status.${cluster?.status}.alert.title`)}
+                        message={cluster?.statusMessage}
+                        variant={cluster.status === ClusterStatus.offline ? 'danger' : 'info'}
+                        noClose
+                        style={{ marginBottom: '12px' }}
+                    />
+                )}
```

Comment:
Can't use the alert provider, because if you use an alert consumer here and try to add alerts, it causes a memory leak and React complains